src/theory/quantifiers/candidate_rewrite_database.cpp

```diff
@@ -32,16 +32,23 @@ namespace cvc5::internal {
 namespace theory {
 namespace quantifiers {
 
+/** Attribute true for rewrites where verification was run  */
+struct RewriteVerifiedAttributeId
+{
+};
+typedef expr::Attribute<RewriteVerifiedAttributeId, bool>
+    RewriteVerifiedAttribute;
```

Comment:
Is this needed? How about using `std::unordered_set<Node> d_verified` instead?

Reply:
Yes, I agree.